wireapp/wire-android-reloaded
# feat: muting conversation list details - part 2 (AR-1043)

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-1043" title="AR-1043" target="_blank"><img alt="Story" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10815?size=medium" />AR-1043</a>  Muting options for conversations
  </td></table>
  <br />
 




----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

https://wearezeta.atlassian.net/browse/AR-1043

### Issues

- Part 1: #523 

This PR adds: 
- The tests coverage to the muting operations
- A refactor to display the current muted setting, on the main options menu

### Testing

#### Test Coverage (Optional)


#### How to Test

You should go and tap into a conversation to display options, and in muting options you'll see the current muted configuration, and if you go to the details of this muting options, you should be able to change it.

### Attachments (Optional)

<img src="https://user-images.githubusercontent.com/5806454/164554886-b7f3a227-fb41-4285-a641-c8dcc51ab763.gif" width="300"/>

https://user-images.githubusercontent.com/5806454/164271545-e34a57f1-c2c7-41be-ba87-a66ae0744a8c.mov

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
